R/axis.R

```diff
@@ -38,4 +38,89 @@ easy_rotate_x_labels <- function(angle = 90, side = c("left", "middle", "right")
     ggplot2::theme(axis.text.x = ggplot2::element_text(angle = angle, hjust = hjust))
 }
 
+#' Easily remove axe(s)
+#'
+#' easy_remove_axes by default removes both axes, but can remove only x or y
+#' if "x" or "y" is given to the 'which' argument
+#' 
+#' easy_remove_x_axis and easy_remove_y_axis remove just the x or y axis,
+#' respectively.
+#'
+#' @md
+#' @param which which axis or axes to remove, by default "both"
+#' @param what what axis components to remove
```

Comment:
@param what axis components to remove (`"ticks"`, `"title"`, `"text"`, or `"line"`)